doc/devel/dependencies.rst

```diff
@@ -0,0 +1,156 @@
+.. _dependencies:
+
+============
+Dependencies
+============
+
+Mandatory dependencies
+======================
+
+When installing through a package manager like ``pip`` or ``conda`` the
+mandatory dependencies are automatically installed. This list is mainly for
+reference.
+
+* `Python <https://www.python.org/downloads/>`_ (>= 3.7)
+* `NumPy <https://numpy.org>`_ (>= 1.16)
+* `setuptools <https://setuptools.readthedocs.io/en/latest/>`_
+* `cycler <https://matplotlib.org/cycler/>`_ (>= 0.10.0)
+* `dateutil <https://pypi.org/project/python-dateutil>`_ (>= 2.7)
+* `kiwisolver <https://github.com/nucleic/kiwi>`_ (>= 1.0.1)
+* `Pillow <https://pillow.readthedocs.io/en/latest/>`_ (>= 6.2)
+* `pyparsing <https://pypi.org/project/pyparsing/>`_ (>=2.2.1)
+
+
+.. _optional_dependencies:
+
+Optional dependencies
+=====================
+
+The following packages and tools are not required but extend the capabilities
+of Matplotlib.
+
```

Comment:
I'd add another layer of subheading for each category of optional installs - Backends, Animation, etc

Reply:
Done.